onetl/db/db_reader/db_reader.py

```diff
@@ -109,8 +112,32 @@ class DBReader(FrozenModel):
 
             Some sources does not support data filtering.
 
+    hwm : type[HWM] | None, default: ``None``
+        HWM class to be used as :etl-entities:`HWM <hwm/column/index.html>` value.
+
+        If you want to use some SQL expression as HWM value, you can pass into ``hwm`` class
+        ``hwm.expression="expression"``, like:
+
+        .. code:: python
+
+            from onetl.hwm import AutoHWM
+
+            hwm = AutoHWM(
+                name="...", column="hwm_column", expression="cast(hwm_column_orig as date)"
```

Comment:
```suggestion
                name="some_unique_hwm_name",
                column="hwm_column",
                expression="cast(hwm_column_orig as date)",
```

В changelog кстати тоже стоит заменить `...` на что-то приближенное к реальности